randao/src/interfaces/IRandao.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;

```

Comment:
change to 0.8.19, we aren't sure higher versions will work

Reply:
lgtm :)